Lib/test/test_typing.py

```diff
@@ -5182,6 +5182,18 @@ class C(B[int]):
                 x = pickle.loads(z)
                 self.assertEqual(s, x)
 
+        # Test ParamSpec args and kwargs
+        global PP
```

Comment:
could this be combined into the `global` statement at the top of the function? (`global C, PP`)

Reply:
Fair enough, I don't feel strongly :)